src/DependencyInjection/ContainerFactory.php

```diff
@@ -28,7 +28,7 @@ public function create(): Container
 
         $container->singleton(Parser::class, static function (): Parser {
             $parserFactory = new ParserFactory();
-            return $parserFactory->create(ParserFactory::PREFER_PHP7);
+            return $parserFactory->createForNewestSupportedVersion();
```

Comment:
per upgrade guide, this should be roughly equivalent

https://github.com/nikic/PHP-Parser/blob/master/UPGRADE-5.0.md#changes-to-the-parser-factory